src/libraries/Microsoft.PowerFx.Core/Public/Config/OptionSet.cs

```diff
@@ -55,7 +56,7 @@ public OptionSet(string name, DisplayNameProvider displayNameProvider)
         /// Contains the members of the option set.
         /// Key is logical/invariant name, value is display name.
         /// </summary>
-        public ImmutableDictionary<DName, DName> Options { get; }
+        public IEnumerable<KeyValuePair<DName, DName>> Options { get; }
```

Comment:
> IEnumerable<KeyValuePair<DName, DName>> [](http://example.com/codeflow?start=14&length=40)

This is a breaking change, but we should go ahead and do it. This is recent code and should be "safe".
We shouldn't have exposed ImmutableDictionary in a public interface in the first place.